src/common/xo.js

```diff
@@ -559,7 +559,9 @@ export const removeTag = (id, tag) => (
 // Backups -----------------------------------------------------------
 
 export const createSchedule = (jobId, cron, enabled) => (
-  xo.call('schedule.create', { jobId, cron, enabled })
+  xo.call('schedule.create', { jobId, cron, enabled }).then(() => {
+    subscribeSchedules.forceRefresh()
+  })
```

Comment:
Avoid *swallowing* the result of the call, you may wanna use this utility pseudo-method:

```js
function tap (cb) {
  return this.then(value => {
    return cb().then(() => value)
  })
}
```